chore(Autolayout): doc update (#615)
unoplatform/uno.toolkit.ui: doc/controls/AutoLayoutControl.md

-|-|-
PrimaryAlignment|AutoLayoutPrimaryAlignment|Either Auto, the child grid take only the place it need in the autolayout direction, or Stretch, the child grid take all the available place in the autolayout direction. Note if set to Strech AutoLayoutJustify will behave as if in Stack mode.
CounterAlignment|AutoLayoutAlignment|Indicates where an element should be displayed on the counter axis of the parent orientation relative to the allocated layout slot of the parent element.

> PrimaryLength|double| Set the grid size in parent orientation. Height or Width (depending of the Orientation) will have priority.

Reviewer: ```suggestion
PrimaryLength|double| Sets the height or width of the child depending on the `Orientation`.
```

Reviewer: I _think_ this is how it should be written, unless I'm misunderstanding how PrimaryLength works

> CounterLength|double|Sets the child size in parent counter orientation. Height or Width (depending of the Orientation) will have priority. 

Reviewer: ```suggestion
CounterLength|double|Sets the height or width of the child depending on the inverse of `Orientation`.
```

Reviewer: I _think_ this is how it should be written, unless I'm misunderstanding how CounterLength works
